src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private int threshold;
+    private int capacity;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+        capacity = DEFAULT_CAPACITY;
+        threshold = (int) (capacity * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size == threshold) {
+            grow();
+        }
+        int bucketIndex = getBucketIndex(key);
+        Node<K, V> node = table[bucketIndex];
+        if (node == null) {
+            table[bucketIndex] = new Node<>(key,value,null);
```

Comment:
```suggestion
            table[bucketIndex] = new Node<>(key, value, null);
```
you have more places like that, check yourself